pyproject.toml

```
@@ -28,5 +29,53 @@ exclude = [
    'IPython/utils/path.py',
    'IPython/utils/timing.py',
    'IPython/utils/text.py'
-   ]
+]
 
+[tool.pytest.ini_options]
+addopts = [
+   "--durations=10",
+   "-p IPython.testing.plugin.pytest_ipdoctest",
```

It looks unhappy and complain about a module names " IPython" (with leading space). Let's try no space here and otherwise lets try to split the string, and `=` 
```suggestion
   "-pIPython.testing.plugin.pytest_ipdoctest",
```